parcels/examples/tutorial_nemo_3D.ipynb

```diff
@@ -0,0 +1,161 @@
+      "P[2](lon=3.516687, lat=51.766171, depth=0.902212, time=345600.000000)\n",
+      "P[3](lon=3.587906, lat=51.770084, depth=0.824910, time=345600.000000)\n",
+      "P[4](lon=3.638291, lat=51.767937, depth=0.761362, time=345600.000000)\n",
+      "P[5](lon=3.326173, lat=51.803127, depth=1.002215, time=345600.000000)\n",
+      "P[6](lon=3.420991, lat=51.801682, depth=0.919173, time=345600.000000)\n",
+      "P[7](lon=3.510124, lat=51.798275, depth=0.876406, time=345600.000000)\n",
+      "P[8](lon=3.585248, lat=51.807373, depth=0.806789, time=345600.000000)\n",
+      "P[9](lon=3.641669, lat=51.818974, depth=0.737283, time=345600.000000)\n",
+      "P[10](lon=3.332299, lat=51.836002, depth=0.991055, time=345600.000000)\n",
+      "P[11](lon=3.423283, lat=51.833839, depth=0.907191, time=345600.000000)\n",
+      "P[12](lon=3.503452, lat=51.829502, depth=0.849505, time=345600.000000)\n",
+      "P[13](lon=3.580498, lat=51.837696, depth=0.792666, time=345600.000000)\n",
+      "P[14](lon=3.645211, lat=51.862221, depth=0.726525, time=345600.000000)\n",
+      "P[15](lon=3.351479, lat=51.872242, depth=0.970860, time=345600.000000)\n",
+      "P[16](lon=3.425382, lat=51.865955, depth=0.895562, time=345600.000000)\n",
+      "P[17](lon=3.499724, lat=51.860241, depth=0.837238, time=345600.000000)\n",
+      "P[18](lon=3.576936, lat=51.867104, depth=0.776531, time=345600.000000)\n",
+      "P[19](lon=3.656613, lat=51.897182, depth=0.721150, time=345600.000000)\n",
+      "P[20](lon=3.356107, lat=51.907131, depth=0.997156, time=345600.000000)\n",
+      "P[21](lon=3.425848, lat=51.899635, depth=0.932838, time=345600.000000)\n",
+      "P[22](lon=3.496043, lat=51.892582, depth=0.865501, time=345600.000000)\n",
+      "P[23](lon=3.567592, lat=51.888668, depth=0.809955, time=345600.000000)\n",
+      "P[24](lon=3.654462, lat=51.919956, depth=0.713223, time=345600.000000)\n"
+     ]
+    }
+   ],
+   "source": [
+    "from parcels import FieldSet, ParticleSet, JITParticle, AdvectionRK4_3D\n",
+    "from glob import glob\n",
+    "import numpy as np\n",
+    "from datetime import timedelta as delta\n",
+    "from os import path\n",
+    "\n",
+    "data_path = 'NemoNorthSeaORCA025-N006_data/'\n",
+    "ufiles = sorted(glob(data_path+'ORCA025-N06_200001??d05U.nc'))\n",
+    "vfiles = sorted(glob(data_path+'ORCA025-N06_200001??d05V.nc'))\n",
+    "wfiles = sorted(glob(data_path+'ORCA025-N06_200001??d05W.nc'))\n",
+    "mesh_mask = data_path + 'coordinates.nc'\n",
+    "\n",
+    "filenames = {'U': {'lon': mesh_mask, 'lat': mesh_mask, 'depth': wfiles[0], 'data': ufiles},\n",
+    "             'V': {'lon': mesh_mask, 'lat': mesh_mask, 'depth': wfiles[0], 'data': vfiles},\n",
+    "             'W': {'lon': mesh_mask, 'lat': mesh_mask, 'depth': wfiles[0], 'data': wfiles}}\n",
+    "\n",
+    "variables = {'U': 'uo',\n",
+    "             'V': 'vo',\n",
+    "             'W': 'wo'}\n",
```

Comment:
Are we sure that `wo` is positive downward, as is assumed by the `AdvectionRK4_3D` kernel? Otherwise, should we add a `set_scaling_factor(-1)`?